discord/highlights.rb

```diff
@@ -1,7 +1,7 @@
 module Highlights
   extend Discordrb::Commands::CommandContainer
 
-  command(:highlights, min_args: 1, max_args: 2) do |event, action, word|
+  command(:highlights, min_args: 1, max_args: 2, description: "Marks a phrase as alertable, and will alert you when that phrase is used.") do |event, action, word|
```

Comment:
Can be modified beyond describing the add function

Reply:
```suggestion
  command(:highlights, min_args: 1, max_args: 2, description: "Use `list` to view alertable phrases you have set up, use `add` and `remove` to add or remove alertable phrases respectively.") do |event, action, word|
```
I think this should be sufficient.  Does the [`] affect the actual ruby file?  If so what would you recommend me change it to?

Reply:
how did you do that

Reply:
just hit the button in the commenting panel, it's the one with the **+** and **-** right before the header text.  (Or press `command + g`)